active/IndexPath/IndexPath.md

```diff
@@ -0,0 +1,96 @@
+# Background
+
+To easily track selection in nested collections, we need an object that we can use to uniquely identify an item in nested collections.
+This can be done with the IndexPath.
+
+# Description
+
+The `IndexPath` class provides indexing of nested collections, for example in a TreeView.
```

Comment:
Is there a corresponding update to TreeView to accept an IndexPath?

Reply:
No. TreeView does not use this yet. This is used by SelectionModel, which still needs an API spec. I would make this public along with SelectionModel.

Reply:
For a feature that cuts across multiple APIs, we likely want a single API spec.

Reply:
Should we/I add the SelectionModel to this API spec since it is the main API this will interact with (and the only one so far).